clusterloader2/pkg/config/cluster.go

```diff
@@ -16,15 +16,18 @@ limitations under the License.
 
 package config
 
+import (
+	"k8s.io/perf-tests/clusterloader2/api"
+)
+
 // ClusterLoaderConfig represents all flags used by CLusterLoader
 type ClusterLoaderConfig struct {
-	ClusterConfig            ClusterConfig `json: clusterConfig`
-	ReportDir                string        `json: reportDir`
-	EnablePrometheusServer   bool          `json: enablePrometheusServer`
-	EnableExecService        bool          `json: enableExecService`
-	TearDownPrometheusServer bool          `json: tearDownPrometheusServer`
-	TestConfigPath           string        `json: testConfigPath`
-	TestOverridesPath        []string      `json: testOverrides`
+	ClusterConfig            ClusterConfig    `json: clusterConfig`
+	ReportDir                string           `json: reportDir`
+	EnablePrometheusServer   bool             `json: enablePrometheusServer`
+	EnableExecService        bool             `json: enableExecService`
+	TearDownPrometheusServer bool             `json: tearDownPrometheusServer`
+	TestScenario             api.TestScenario `json: testScenario`
```

Comment:
Wait - TestScenario is not a flag to ClusterLoader (it's the path to config file with test scenario)

Reply:
"testSuiteConfigPath" - flag - Path to the test suite config file.
"TestScenario" - structure - defines test config, overrides and identifier.